### PR TITLE
[CI] Stabilize visual tests, remove appAction, and update pane buttons

### PR DESCRIPTION
**Pull request by unlikelyzero**
Note: this is blocked on https://github.com/nasa/openmct/pull/6990

### Describe your changes:
- Adds the 'timer create date' css to percyCSS to ignore a changing timestamp on notebooks. Note: we may want to remove this test if it adds no value
- Adds a new constant called `VISUAL_URL` which contains all of the encoded paths necessary to hide the inspector and the tree and put the clock into fixed time
- Updates documentation
- Adds pane button name attribute to open inspector and tree panes
- Removes appAction selectInspectorTab

### All Submissions:

* [x] Have you followed the guidelines in our [Contributing document](https://github.com/nasa/openmct/blob/master/CONTRIBUTING.md)?
* [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/nasa/openmct/pulls) for the same update/change?
* [x] Is this change backwards compatible? For example, developers won't need to change how they are calling the API or how they've extended core plugins such as Tables or Plots.

### Author Checklist

* [x] Changes address original issue?
* [x] Tests included and/or updated with changes?
* [x] Command line build passes?
* [x] Has this been smoke tested?
* [x] Testing instructions included in associated issue OR is this a dependency/testcase change?

### Reviewer Checklist

* [ ] Changes appear to address issue?
* [ ] Reviewer has tested changes by following the provided instructions?
* [ ] Changes appear not to be breaking changes?
* [ ] Appropriate automated tests included?
* [ ] Code style and in-line documentation are appropriate?
* [ ] Has associated issue been labelled unverified? (only applicable if this PR closes the issue)
* [ ] Has associated issue been labelled bug? (only applicable if this PR is for a bug fix)

### CODE REVIEW

File: e2e/tests/visual/notification.visual.spec.js [hunk 1]
```diff
@@ -27,11 +27,12 @@
 const { test, expect } = require('../../pluginFixtures');
 const percySnapshot = require('@percy/playwright');
 const { createDomainObjectWithDefaults } = require('../../appActions');
+const VISUAL_URL = require('../../constants').VISUAL_URL;
 
 test.describe("Visual - Check Notification Info Banner of 'Save successful'", () => {
   test.beforeEach(async ({ page }) => {
-    // Go to baseURL and Hide Tree
-    await page.goto('./', { waitUntil: 'networkidle' });
+    //await page.goto('./');
```
Comment (scottbell) [inline e2e/tests/visual/notification.visual.spec.js:34]: we should delete the commented out code

**Review by scottbell — COMMENTED** [review]
One small change, but looks good otherwise!

File: e2e/tests/visual/controlledClock.visual.spec.js [hunk 2]
```diff
@@ -25,35 +25,31 @@ Collection of Visual Tests set to run with browser clock manipulate made possibl
...
 test.describe('Visual - Controlled Clock', () => {
   test.beforeEach(async ({ page }) => {
-    //Go to baseURL and Hide Tree
-    await page.goto('./#/browse/mine?hideTree=true', { waitUntil: 'networkidle' });
+    await page.goto(VISUAL_URL, { waitUntil: 'domcontentloaded' });
   });
   test.use({
     storageState: './e2e/test-data/overlay_plot_with_delay_storage.json',
     clockOptions: {
+      now: MISSION_TIME,
```
Comment (unlikelyzero) [inline e2e/tests/visual/controlledClock.visual.spec.js:39]: driveby to match the readme

**Review by ozyx — APPROVED** [review]
LGTM! Just had a question about the prettier bypass

File: e2e/constants.js [hunk 3]
```diff
@@ -1,3 +1,4 @@
+/* eslint-disable prettier/prettier */
```
[the remark this answers is not in the document]
Comment (unlikelyzero) [reply e2e/constants.js:1]: the length of the constants are too long. I figured it'd be a regular thing if we're doing URLs

[the remark this answers is not in the document]
Comment (ozyx) [reply e2e/constants.js:1]: oh, wild. I didn't know that would fail prettier? does it not except that automatically?
